helm/scylla-operator/templates/scyllacluster_member_clusterrole_def.yaml

```diff
@@ -47,6 +47,8 @@ rules:
   - list
   - patch
   - watch
+# Sidecar no longer fetches ScyllaClusters, as necessary information is provided by the Operator to sidecar via arguments.
```

Comment:
nit: add a TODO to make this stand out a bit more (an issue as well?)

Reply:
done